src/data/recipies.json

```
@@ -30,18 +27,59 @@
         "amount": 2
       }
     ],
+    "instructions": ["Każdy umie zrobić jajecznicę."],
+    "developedBy": "Przemysław Onak"
+  },
+  {
+    "id": "jajecznica-z-kielbasa",
+    "name": "Jajecznica z kiełbasą",
+    "description": "Pożywna i klasyczna wersja jajecznicy, idealna na solidne śniadanie lub szybką kolację.",
+    "mealType": ["śniadanie", "kolacja"],
+    "difficulty": 2,
+    "ingredients": [
+      {
+        "id": "jajka",
+        "amount": 2
+      },
+      {
+        "id": "kielbasa",
+        "amount": 75
+      },
+      {
+        "id": "cebula",
+        "amount": 0.25
+      },
+      {
+        "id": "maslo",
+        "amount": 10
+      },
+      {
+        "id": "sol",
+        "amount": 1
+      },
+      {
+        "id": "pieprz",
+        "amount": 1
+      },
+      {
+        "id": "chleb",
+        "amount": 0.3
+      }
+    ],
     "instructions": [
-      "Każdy umie zrobić jajecznicę."
+      "Kiełbasę i cebulę pokrój w drobną kostkę.",
+      "Na patelni rozgrzej masło, zeszklij cebulę, a następnie dodaj kiełbasę i smaż do zarumienienia.",
+      "W misce roztrzep jajka z solą i pieprzem.",
+      "Wlej masę jajeczną na patelnię i smaż, mieszając, aż jajka osiągną pożądaną konsystencję.",
```

```suggestion
      "Wlej masę jajeczną na patelnię i smaż, mieszając, aż jajka osiągną pożądaną konsystencję.",
```
Część osób nie lubi jajecznicy, która nie jest ścięta, ale też nie powinna mieć konsystencji podeszwy od buta.